ledger/jetcoordinator/jetcoordinator.go

```diff
@@ -107,7 +109,23 @@ func (jc *JetCoordinator) getNextValidators(candidates [][]byte, count int) ([][
 	return selected, nil
 }
 
+func (jc *JetCoordinator) jetRef(objRef core.RecordRef) *core.RecordRef { // nolint: megacheck
+	return jc.rootJetNode.GetContaining(&objRef)
+}
+
 // NewJetCoordinator creates new coordinator instance.
 func NewJetCoordinator(db *storage.DB) (*JetCoordinator, error) {
-	return &JetCoordinator{db: db}, nil
+	// TODO: temporary jet tree (later will be fetched fom config or ledger)
+	rootJetNode := &JetNode{
```

Comment:
Same thing as below. Not all tree nodes are Jets

Reply:
They used here as thresholds. If node doesn't have children it leaf, then we use it as ref. In other cases we treat it like splitting value.